### PR TITLE
add send email flag

### PR DESCRIPTION
**Pull request by rmudambi**

### CODE REVIEW

File: src/covid_shared/cli_tools.py [hunk 1]
```diff
@@ -58,6 +58,10 @@ def configure_logging_to_files(output_path: Path) -> None:
     click.option('-v', 'verbose',
                  count=True,
                  help='Configure logging verbosity.'),
+    click.option('-e, --email-alert',
+                 type=bool,
+                 default=False,
+                 help='Send email when final job is done')
```
Comment (miker985) [inline src/covid_shared/cli_tools.py:64]: Doesn't this decorator need `is_flag=True`?

You don't provide an option to specify the recipient(s). Is the plan to hard-code just the calling user? That will probably work fine, but I'd suggest adding that information to the `--email-alert` help text.

Comment (rmudambi) [reply src/covid_shared/cli_tools.py]: You're completely right. Fixed that and updated the help text.

File: src/covid_shared/cli_tools.py [hunk 2]
```diff
@@ -58,6 +58,10 @@ def configure_logging_to_files(output_path: Path) -> None:
     click.option('-v', 'verbose',
                  count=True,
                  help='Configure logging verbosity.'),
+    click.option('-e, --email-alert',
+                 is_flag=True,
+                 default=False,
+                 help='Send email to the calling user when final job is done')
 ]
```
Comment (rmudambi) [inline src/covid_shared/cli_tools.py:67]: Not sure the best way to rename this function in a backwards compatible way

Comment (JosephWagner) [reply src/covid_shared/cli_tools.py:67]: We could rename it with a more generic name, and then create a new function with the same original name that A) raises a deprecation warning explaining that callers should use the renamed function and B) calls the renamed function.

There's only what, like 4 packages using this? We could update each individually over the next week and then delete the deprecated function

**Comment by rmudambi** [comment]
Tabling this for now.

**Pull request closed without merging**